### DISCUSSION TITLE
Question on simulation stability

### DISCUSSION THREAD

**Discussion by jungtaekkim** [NanoComp/meep#2668] [Q&A] [answered]
Is there any difference on simulation stability between simulations with a unit length of 10 nm and a resolution of 10 and ones with a unit length of 1 nm and a resolution of 1?  I think the numbers of pixels for both simulations are identical.

In my simulations, the setting with a unit length of 10 nm and a resolution of 10 is more stable.

More specifically, the simulations with a unit length of 1 nm and a resolution of 1 sometimes get stuck before starting the simulation.  All these simulations are run on shared-memory systems.  Can this issue be related to the [issue linked](https://meep.readthedocs.io/en/latest/FAQ/#why-does-the-time-stepping-rate-fluctuate-erratically-for-jobs-running-on-a-shared-memory-system)?  On the contrary, I did not encounter any issue with the simulations with a unit length of 10 nm and a resolution of 10.

**Comment by smartalecH** [comment, marked as answer]
The $\Delta t$ will be different between the two simulations, so it makes sense that one is more stable than the other.
